### ISSUE TITLE
WebViewer RTL UI (zoom buttons)

### ISSUE THREAD

**Issue by zartom** [ericberman/MyFlightbookAndroid#275]
![Screenshot_20220603-113030_MyFlightbook](https://user-images.githubusercontent.com/1769078/171819322-0031c756-e10f-47c4-b3d7-2e08cc1829ff.jpg)

**Comment by zartom** [comment]
AFAIK all RTL issues in android (since Android 6 or something) are solved by using "start" and "end" layout attributes instead of "left" and "right" like in the early days. Then the system decides what to do with the UI in most contexts.

**Comment by ericberman** [comment]
Indeed, and I'm doing start/end elsewhere.  But alas, my point above: the layout for those zoom buttons is not anywhere in my code.  I instantiate a (system-provided) webview, and the webview draws the zoom controls.  So I don't have any place to put "start" or "end."  I looked through the docs and can't find any way to control those; the webview just shows/hides them automatically.

Are the top toolbar buttons (refresh/forward/back) "correct" in the sense of being logical for rtl?  Those are doing start/end (implicitly - they're just inside a linear layout and the linear layout is doing the ltr/rtl reversal)

**Comment by ericberman** [comment]
So anyhow, I have no idea how to solve the zoom buttons issue.  That appears to be at the OS level.

**Comment by ericberman** [comment]
Looks like someone else did their own zoom buttons because indeed it's an android bug.
https://github.com/osmdroid/osmdroid/issues/1053
